docs/src/index.md

```diff
@@ -299,47 +299,51 @@ function foo(x)
 end
 
 #### Find dfoo/dx via rrules
-
-# First the forward pass, accumulating rules
+#### First the forward pass, accumulating rules
 x = 3;
```

Comment:
```suggestion
 x = 3
```

And below as well.

Reply:
This prevents output when you copy paste into the repl

Which I think is nice

Reply:
Lets keep it in then :)